Week1/REAL_WORLD.md

```diff
@@ -0,0 +1,33 @@
+# Shared testing activities for web and mobile (Real World App)
```

Comment:
Amazing Details Dua 👍 ! We mentioned that as QAs you should be able to zoom in and out, so it would be extra great if you can zoom out to one level up and group activities for instance *functional testing* would include testing all the functionalities that you mentioned. 
And for mobile, maybe also add the things we discussed; such as gestures, receiving calls and different types of network connections